migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/upgrade.yaml

```diff
@@ -233,17 +244,6 @@
     target: "{{ astro_save_dir.path }}/astronomer-db-backup.tar"
     target_opts: "--clean --create --format=tar"
 
-- name: Run SQL for Prisma 1 to Prisma 2 migration
-  register: prisma1_to_2
-  postgresql_query:
-    port: "{{ db_port }}"
-    autocommit: yes
-    login_host: "{{ db_hostname }}"
-    login_user: "{{ db_username }}"
-    login_password: "{{ db_password }}"
-    db: "{{ database_name }}_houston"
-    path_to_script: "files/prisma1_to_prisma2.sql"
-
```

Comment:
@danielhoherd have you confirmed steps 2 and 3 below for the 0.23 to 0.25 upgrade? I dont think there were any Houston data migrations done between 0.23 and 0.25

Reply:
This file no longer exists.

Reply:
but you have moved it to 0.23-to-0.25/playbooks/tasks/upgrade.yaml right? And retaining all the 6 steps of the upgrade process?

Reply:
I am still working on getting tests to pass. I haven't tuned the flow yet, but you're right, it looks like steps 2 and 3 can be deleted. Thanks for catching that!

Reply:
@rbankston @andriisoldatenko - please review the steps below and confirm which ones are necessary for 0.23 to 0.25 upgrade

Reply:
If there isn't a database change how does https://github.com/astronomer/issues/issues/2920 get fixed?

Reply:
This test case is not caught by our upgrade testing because it requires an existing deployment to trigger the failure.